### PR TITLE
Update run directory scripts to get certain restarts from 14.3.0 1-year benchmark output and boundary condition output

### PR DESCRIPTION
**Pull request by yantosca**
### Name and Institution (Required)

Name: Bob Yantosca
Institution: Harvard + GCST

### Confirm you have reviewed the following documentation

- [x] [Contributing guidelines](https://geos-chem.readthedocs.io/en/stable/help-and-reference/CONTRIBUTING.html)

### Describe the update

**NOTE: This PR should only be merged just before the 14.3.0 release!**

This PR updates the `run/shared/download_data.yml` config file and `run/GCHP/createRunDir.sh` script so that the fullchem and TransportTracers restart files will be copied from the `ExtData/GEOSCHEM_RESTARTS/GC_14.3.0` folder.  This folder contains restarts from the 1-year fullchem and TransportTracers benchmarks.

### Expected changes
This is necessary structural update that will prevent GEOS-Chem simulations from failing "out-of-the-box" due to missing species in the restart files.  This modification is also necessary to pass no-bootstrap integration tests.

### Reference(s)
N/A

### Related Github Issue(s)
N/A

### CODE REVIEW

**Comment by yantosca** [comment]
The fullchem nested files are failing due to missing species in the boundary conditions files.
```console
HEMCO: Opening /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/HEMCO/SAMPLE_BCs/v2021-09/fullchem/GEOSChem.BoundaryConditions.20190701_0000z.nc4
 
HEMCO ERROR: Cannot find field BC_BUTDI. Please check variable name in the config. file
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Increse requested time limits in GCHP integration tests (compile 2h30m, run 5h)
 - Changed CO2 concentration used in RRTMG to be modifiable in geoschem_config.yml
 - Changed water vapor used in RRTMG to match to tracer field at all altitudes
+- Look for fullchem and TransportTracers restarts in the `GEOSCHEM_RESTARTS/GC_4.3.0` folder
```
Comment (lizziel) [inline CHANGELOG.md:33]: This should be 14.3.0.

Comment (yantosca) [reply CHANGELOG.md:33]: Thanks Lizzie!

Comment (yantosca) [reply CHANGELOG.md]: Now fixed in commit f637f11

**Comment by yantosca** [comment]
@msulprizio @lizziel: I've also re-generated the fullchem BoundaryConditions collection output from a 1-day 4x5 run with 14.3.0.  This should now have all species and should pass no-bootstrap integration tests.

**Review by msulprizio — APPROVED** [review]

**Review by msulprizio — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,6 +30,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Increse requested time limits in GCHP integration tests (compile 2h30m, run 5h)
 - Changed CO2 concentration used in RRTMG to be modifiable in geoschem_config.yml
 - Changed water vapor used in RRTMG to match to tracer field at all altitudes
+- Look for fullchem and TransportTracers restarts in the `GEOSCHEM_RESTARTS/GC_14.3.0` folder
```
Comment (lizziel) [inline CHANGELOG.md:33]: Update to only include fullchem since TransportTracers will be GC_14.2.0

**Review by lizziel — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (yantosca) [reply CHANGELOG.md]: Now fixed in commit 8adbf0096

**Review by lizziel — APPROVED** [review]
